src/forecast.coffee

```diff
@@ -4,13 +4,15 @@
 # Configuration:
 #   HUBOT_FORECAST_DAYS - When do you need weather? Default: mon,tue,wed,thu,fri
 #   HUBOT_FORECAST_KEY - forecast.io API key
-#   HUBOT_FORECAST_ROOM - Room bulletins should be posted in
+#   HUBOT_FORECAST_ROOM - Room bulletins should be posted in (if slack, just use channel, no #)
```

Comment:
I coulda sworn you use the `#` symbol… would you point me to the docs on this? Want to be sure we get this right!